src/net_protocol.rs

```diff
@@ -66,6 +66,14 @@ pub struct BlobsProtocol {
     pub(crate) inner: Arc<BlobsInner>,
 }
 
+impl Deref for BlobsProtocol {
+    type Target = Store;
+
+    fn deref(&self) -> &Self::Target {
+        &self.inner.store
+    }
```

Comment:
Personally I'm not a huge fan of derefs, but also 🤷

Reply:
> Personally I'm not a huge fan of derefs, but also 🤷

If we don't like derefs in public APIs, we would also have to remove the deref from MemStore and FsStore to the client API. But if we want to do this, it would have to be a separate PR since it would affect a lot of places.